caps/templates/council_detail.html

```diff
@@ -77,6 +77,24 @@ <h2 class="mr-auto pr-3 mb-1 mb-sm-0">Climate documents</h2>
                 <small class="text-muted line-height-1">Last update: {{ last_updated.last_update }}</small>
               {% endif %}
             </div>
+
+            {% if council.plandocument_set.all %}
+            <div class="card-body is-emphasised-section">
+                <form action="{% url 'search_results' %}" data-show-interstitial class="form-row flex-wrap align-items-center">
+                    <div class="col-auto flex-shrink-1 flex-grow-1" style="max-width: 450px">
+                        <input type="text" class="form-control form-control-sm" name="q" id="search-this-councils-documents" value="{{ q|default_if_none:'' }}" placeholder="Search inside these documents">
```

Comment:
This input needs a `<label>`, rather than hiding label text in a `placeholder` attribute. We could make the `label` screenreader-only, though, if you like, to keep the UI minimal.

Wonder whether we should give it `type="search"` too, while we’re at it…